CityOfDetroit/bloom
# 1373/housing basics page

# Pull Request Template

## Issue Overview

This PR addresses #1373


## Description

This PR brings over the media card, video component, and updated modal and implements them on the new housing-basics page. It also updates the action blocks at the bottom of the home page to include a new section and updated icons so that all three icons are aligned.

## How Can This Be Tested/Reviewed?

This PR can be tested on Detroit Public's site. After arriving at the homepage, scroll down to the action blocks and ensure the new section is correct and the three sections are aligned. Then click learn more and test out the functionality of the media cards by clicking on the title or any part of the green header on the card. After you've tested viewing the video, ensure the modal can be closed and others cards opened.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202653933913055
